.github/workflows/tests.yml

```
@@ -10,21 +10,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-version: ['7.4', '8.0', '8.1', '8.2']
+        # Symfony 6.3 requires PHP >= 8.0, it will be installed on PHP >= 8.0
+        # Symfony 7.0 requires PHP >= 8.2, it will be installed on PHP >= 8.2
+        php-version: ['8.1', '8.2']
         composer-flags: ['']
-        symfony-version: ['^5.4']
+        symfony-version: ['^6.4']
         include:
-          - php-version: 7.4
-            symfony-version: "^4.4"
-            composer-flags: "--prefer-lowest"
-          - php-version: 7.4
-            symfony-version: "^4.4"
-          - php-version: 8.0
-            symfony-version: "^4.4"
           - php-version: 8.1
```

```suggestion
          - php-version: 7.4
            symfony-version: "^5.4"
            composer-flags: "--prefer-lowest"
          - php-version: 8.1
            symfony-version: "^5.4"
          - php-version: 8.1
```

Let's keep the tests with Symfony 5.4.

---

Ok, I was pretty sure you'll say something like this 👍